gui/src/components/WorkspaceNav.svelte

```diff
@@ -0,0 +1,82 @@
+<script lang="ts">
+  import * as router from 'svelte-spa-router';
+
+  import Home from './mono/home.svelte';
+  import Layers from './mono/layers.svelte';
+  import Storage from './mono/database.svelte';
+  import Networking from './mono/networking.svelte';
+
+  export let workspaceId: string;
+  export let active: string;
+</script>
+
+<aside>
+  <button
+    on:click={() => {
+      router.push(`#/workspaces/${workspaceId}`);
+    }}
+    class={active === 'Dashboard' ? 'active' : ''}
+    title="Dashboard"><Home /></button
+  >
+  <button
+    on:click={() => {
+      router.push(`#/workspaces/${workspaceId}/components`);
+    }}
+    class={active === 'Components' ? 'active' : ''}
+    title="Components"><Layers /></button
+  >
+  <button
+    on:click={() => {
+      router.push(`#/workspaces/${workspaceId}/storage`);
+    }}
+    class={active === 'Storage' ? 'active' : ''}
+    title="Storage"><Storage /></button
+  >
+  <button
+    on:click={() => {
+      router.push(`#/workspaces/${workspaceId}/networking`);
+    }}
+    class={active === 'Networking' ? 'active' : ''}
+    title="Networking"><Networking /></button
+  >
```

Comment:
is this how prettier formats these? yuck.

Reply:
Yes, but it can be fixed, which I will do.